docs/contributing/building/building.md

```diff
@@ -58,14 +58,14 @@
         $ make clean-bpf-nocore     	# wipe ./dist/tracee.bpf.XXX.o
         $ make clean-tracee-ebpf    	# wipe ./dist/tracee-ebpf
         $ make clean-tracee-rules   	# wipe ./dist/tracee-rules
-        $ make clean-rules          	# wipe ./dist/rules
+        $ make clean-signatures       # wipe ./dist/signatures
 
     # test
 
         $ make test-types           	# run unit tests for types module
         $ make test-unit            	# run unit tests
         $ make test-integration     	# run integration tests
-        $ make test-rules           	# opa test (tracee-rules)
+        $ make test-signatures        # opa test (tracee-rules)
```

Comment:
nit: unaligned, there are tabs and spaces and that might lead to unalignment. I would replace all tabs for spaces (in between the target name and the # comment).